src/components/Bar.jsx

```diff
@@ -189,8 +190,17 @@ class Bar extends Component {
       onLogOut,
       userActionRequired
     } = this.props
+
+    const { 
+      primaryColor: pColor, 
+      primaryContrastTextColor: pctColor 
+    } = themeOverrides
+    const pStyle = pColor ? { '--cozBarThemePrimaryColor': pColor } : { }
+    const pctStyle = pctColor ? { '--cozBarThemePrimaryContrastTextColor': pctColor } : { }
+    const themeStyle = { ...pStyle, ...pctStyle }
```

Comment:
```
    const pStyle = pColor ? { '--cozBarThemePrimaryColor': pColor } : null
    const pctStyle = pctColor ? { '--cozBarThemePrimaryContrastTextColor': pctColor } : null
    const themeStyle = pStyle || pctStyle ? { ...pStyle, ...pctStyle } : null
```

This way, when pColor and and pctStyle is empty, we do not put nothing in style. See the snaps below with lots of empty `style`.